frontend/src/core/user/components/UserMenu.js

```diff
@@ -136,39 +136,45 @@ export class UserMenuBase extends React.Component<Props, State> {
 
                 <li>
                     <Localized
-                        id="user-UserMenu--top-contributors"
+                        id="user-UserMenu--terms"
                         glyph={
-                            <i className="fa fa-trophy fa-fw"></i>
+                            <i className="fa fa-gavel fa-fw"></i>
                         }
                     >
-                        <a href="/contributors/">
-                            { '<glyph></glyph>Top Contributors' }
+                        <a href="/terms/"
+                        rel="noopener noreferrer"
+                        target="_blank">
```

Comment:
Nit: We try to indent attributes like this:
```html
<a
    href="/terms/"
    rel="noopener noreferrer"
    target="_blank"
>
```
Please fix that for all four links: